compiler/backend/src/org/jetbrains/kotlin/codegen/FunctionCodegen.java

```diff
@@ -647,7 +647,19 @@ else if (isCompatibilityStubInDefaultImpls(functionDescriptor, context, jvmDefau
                     KtDeclarationWithBody declaration = ((ClosureGenerationStrategy) strategy).getDeclaration();
                     BindingContext bindingContext = typeMapper.getBindingContext();
                     if (declaration instanceof KtFunctionLiteral && isLambdaPassedToInlineOnly((KtFunction) declaration, bindingContext)) {
-                        lineNumber = LOCAL_VARIABLE_INLINE_ARGUMENT_SYNTHETIC_LINE_NUMBER;
```

Comment:
@yanex might be a weird question, but are you sure this ever worked? I tried debugging the following code in IDEA with the 1.4-M1 Kotlin plugin:
```
fun main(args: Array<String>) {
    forEach { print2("123") }
}
fun print2(s: String){
    println(s)
}
@Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
@kotlin.internal.InlineOnly
inline fun forEach(s: () -> Unit) {
    for (i in 1..2) {
        s()
    }
}
```
and a breakpoint on the lambda is never triggered. Recompiling the file manually using the current master kotlinc does not help, but recompiling it with this patch does.

Reply:
The last time I tried it, it worked. However, I don't have anything against your version. The main reasoning with the introduction of 65100 was to slightly reduce the generated lines count.